README.md

```diff
@@ -217,6 +217,14 @@ Client certificates are an alternative to traditional authentication mechanisms.
 
 Newman supports SSL client certificates, via the following CLI options (available with Newman `v3` style `run` only):
 
+#### Use a ssl client certificates configuration file (supports multiple certificates per run)
+
+- --ssl-client-certs<br/>
+The path to the ssl client certs configuration file (json format). See [examples/multiple-ssl-client-certs.json](https://github.com/postmanlabs/newman/blob/develop/examples/multiple-ssl-client-certs.json)
```

Comment:
We should add an example (examples/ssl-client-cert-list.json) as part of this PR.

Reply:
Good catch. Will add this file.